src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -258,6 +258,13 @@ export const thorchainSaversStakingOpportunitiesUserDataResolver = async ({
       let maturity: number | undefined = undefined
 
       if (stakingOpportunityId === thorchainAssetId && accountPosition.last_add_height) {
+      }
+
+      const maybeMaturity = await (async () => {
+        if (stakingOpportunityId !== thorchainAssetId) return {}
+        // note undefined over missing property here, i.e this is a runepool position and explicitly undefined
```

Comment:
rm me

Reply:
fixed in [60e41ac](https://github.com/shapeshift/web/pull/7472/commits/60e41aceb721592767aac449c8b1671168120425)